docs/proton_c_conversion.md

```diff
@@ -8,14 +12,68 @@ You can add the same argument to your keymap's `rules.mk`, which will accomplish
 
 This exposes the `CONVERT_TO_PROTON_C` flag that you can use in your code with `#ifdef`s, like this:
 
-    #ifdef CONVERT_TO_PROTON_C
-        // Proton C code
-    #else
-        // Pro Micro code
-    #endif
+```c
+#ifdef CONVERT_TO_PROTON_C
+    // Proton C code
+#else
+    // Pro Micro code
+#endif
+```
 
-Before being able to compile, you may get some errors about `PORTB/DDRB`, etc not being defined, so you'll need to convert the keyboard's code to use the [GPIO Controls](internals_gpio_control.md) that will work for both ARM and AVR. This shouldn't affect the AVR builds at all.
+If you get errors about `PORTB/DDRB`, etc not being defined, so you'll need to convert the keyboard's code to use the [GPIO Controls](internals_gpio_control.md) that will work for both ARM and AVR. This shouldn't affect the AVR builds at all.
 
 The Proton C only has one on-board LED (C13), and by default, the TXLED (D5) is mapped to it. If you want the RXLED (B0) mapped to it instead, add this like to your `config.h`:
 
     #define CONVERT_TO_PROTON_C_RXLED
+
+## Manual Conversion
+
+To use the Proton C natively, without having to specify `CTPC=yes`, you need to change the `MCU` line in `rules.mk`:
+
+```
+MCU = STM32F303
+```
+
+Remove these variables if they exist:
+
+* `BOOTLOADER`
+* `LINK_TIME_OPTIMIZATION_ENABLE`
+* `EXTRA_FLAGS`
```

Comment:
I mean, these don't need to be removed...  But they aren't really needed. 

But I can verify that they do work.